### DISCUSSION TITLE
[M-Z] (2024/25) - sezione 0.9 es 4.53 (11 novembre)

### DISCUSSION THREAD

**Discussion by AglaiaNorza** [sapienzastudentsnetwork/calcolo-delle-probabilita#321] [Esercizi M-Z]
![es53](https://github.com/user-attachments/assets/f1a45a98-0deb-4a2e-86c7-3cbd6bb79a34)

**Comment by AglaiaNorza** [comment]
potrebbe avere senso
 
![es](https://github.com/user-attachments/assets/0f15ac1e-d4bf-4a61-81c2-ae7d5e9fc19d)

**Comment by marcoseverini** [comment]
![immagine](https://github.com/user-attachments/assets/78a487c2-00ef-4f85-adf2-4df905bff8f2)
